docs/hub/model-repos.md

```diff
@@ -293,18 +293,23 @@ zLib License	| `zlib`
 Open Data Commons Public Domain Dedication and License | `pddl`
 Lesser General Public License For Linguistic Resources | `lgpl-lr`
 
-## How is the carbon footprint of my model calculated?
+## How is the carbon footprint of my model calculated? 🌎
 
 By taking into account the computing hardware, location, usage and training time, it's possible to provide an estimate of how much CO<sub>2</sub> was produced by the model.
 
-The math is actually pretty simple! 
+The math is actually pretty simple! ➕
 
-First, you take the *carbon intensity* of the electric grid that is being used for the training -- this is how much CO<sub>2</sub> is produced by KwH of electricity used. This depends on the location where the hardware is located and the [energy mix](https://electricitymap.org/) used at that location -- whethere it's renewable energy like solar, wind and hydro, or non-renewable energy like coal and natural gas. The more renewable energy is used, the less carbon intensive it is!
-
-Then, you take the power consumption of the GPU  during training -- this is done using the`pynvml` library.
+First, you take the *carbon intensity* of the electric grid that is being used for the training -- this is how much CO<sub>2</sub> is produced by KwH of electricity used. This depends on the location where the hardware is located and the [energy mix](https://electricitymap.org/) used at that location -- whethere it's renewable energy like solar 🌞, wind 🌬️ and hydro 💧, or non-renewable energy like coal ⚫ and natural gas 💨. The more renewable energy is used, the less carbon intensive it is!
+ 
+Then, you take the power consumption of the GPU  during training -- this is done using the `pynvml` library.
 
 Finally, you multiply the power consumption and carbon intensity by the training time of the model, and you have an estimate of how much CO<sub>2</sub> was emitted.
 
-Keep in mind that this isn't an exact number, because there are other factors that come into play -- like the energy that's used for datacenter heating and cooling -- which will increase that number. But this will already give you a good idea of the scale of CO<sub>2</sub> emissions that your model is procuding!
+Keep in mind that this isn't an exact number, because there are other factors that come into play -- like the energy that's used for datacenter heating and cooling -- which will increase that number. But this will already give you a good idea of the scale of CO<sub>2</sub> emissions that your model is producing!
```

Comment:
```suggestion
Keep in mind that this isn't an exact number, because there are other factors that come into play -- like the energy that's used for data center heating and cooling -- which will increase carbon emissions. But this will already give you a good idea of the scale of CO<sub>2</sub> emissions that your model is producing!
```